sources/README.md

```diff
@@ -1525,6 +1525,20 @@ _The Museum Universe Data File is an evolving list of museums and related organi
 * url: _https://www.imls.gov/research-evaluation/data-collection/museum-universe-data-file_
 * license: ``
 
+## Mexico's Comision Nacional de Areas Naturales Protegidas
+
+_Mexico's National database of Biosphere Reserves, National Parks, Natural Monuments, Natural Resources Protection Areas, Flora and Fauna Protection Areas, CONANP Sanctuaries_ 
+
+* added: `2021-09-01`
+* id: `1746277021`
+* name: `mx-conanp`
+* prefix: `mx-conanp`
+* license_type: _CC BY (equivalent)_
+* license_text: _You can: Make and distribute copies of the data set and its content; Disseminate and publish the data set and its content; Adapt or rearrange the data set and its content; Extract all or part of the content of the data set; Commercially exploit the data set and its content, and; Create data sets derived from the data set or its content._
```

Comment:
Just like the Canada source, please prefix with the appropriate credit, in this case:

> "Reservas de la Biosfera, Parques Nacionales, Monumentos Naturales, Áreas de Protección de Recursos Naturales, Áreas de Protección de Flora y Fauna, Santuarios" published by CONANP. Consulted at https://datos.gob.mx/busca/dataset/informacion-espacial-de-las-areas-naturales-protegidas-181 on 2021-09-01.

Reply:
fixed in 29a7daa